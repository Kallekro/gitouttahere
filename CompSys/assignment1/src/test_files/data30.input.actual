test_files/data30.input: data
