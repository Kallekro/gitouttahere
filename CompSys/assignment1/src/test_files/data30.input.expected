test_files/data30.input: very short file (no magic)
